src/main/resources/db/migration/oracle/V1.3__create_table_for_texts.sql

```diff
@@ -0,0 +1,7 @@
+create table texts
+(
+    key          varchar2(255) not null,
+    value        clob,
+    language     char(2)       not null,
+    fromDateTime char(19)      not null
```

Comment:
Endre til fra_dato date

Reply:
Det er bedre å ha varchar2 her nå. Eller må vi ha forskjellige SQL for å håndtere datetime både i Oracle og PostgreSQL.
Vi kan endre dette etter at vi flytter til meldekort-api sin DB.

Reply:
Syntaksen for `date` er lik i både Oracle og PostgreSQL. En tekst gjelder vel for en dato, ikke et bestemt klokkeslett.

```
https://extendsclass.com/postgresql-online.html
create table test (fra_dato date);

insert into test(fra_dato) values(to_date('01.01.2022', 'DD.MM.YYYY'));

select * from test;
```

Reply:
Vel, jeg har endret til date, men dette skaper problemer nå. Det er ikke mulig å ha 0000-00-00

Reply:
Måtte endre til 1000-01-01